docs/src/installation/raspberry_pi_installation.md

```diff
@@ -35,6 +35,10 @@ Once the OS installation has been completed, you are ready to commence.
 
     Follow [instructions for Ubuntu and Debian based distributions like Raspberry Pi OS at NodeSource Distributions](https://github.com/nodesource/distributions#installation-instructions).
 
+   NOTE: IMPORTANT - installing a higher version of nodejs (eg V20) is known to cause issues with a number of plugins and CAN adaptors.
+   Note: Some applications that are installed after signalk-server  are known to "auto" install updated/most recent version of nodejs... 
+    which as above breaks things .... take care to avoid this!
+
```

Comment:
Sorry, I think there are multiple issues in this addition:
- The previous step calls for installing NodeJS (note the capitalization) 18 and this about "higher version" and refers to V20. To me this is needlessly confusing
- Why is there one NOTE and and another note?
- I am not aware of other applications updating nodejs version - can you be more specific? And how would an end user _take care to avoid thid_?

This document is about installing signalk-server on a system that has no NodeJS or a previous installation, but the notes are about updating Node. Maybe we could find some other place where these notes would have a more appropriate home, like https://github.com/SignalK/signalk-server/wiki/Installing-and-Updating-Node.js ?

Reply:
ok noted i see your point.

I did add this into the FAQ, hopefully thats ok? 
![Uploading IMG_0161.jpeg…]()

Reply:
i did follow the instructions as presented, all good.
however when installing either influx or grafana - i believe it was the latter, 
the install script - went ahead and update nodejs… (to V20) … this created two days of fault finding and unnecessary debugging with help from other signalk users. 

FAQs would at best be  looked at after a user encounters an issue … i was thinking “how” to make sure that as part of the install process .. that a user be alerted to beware of this …

Reply:
i did add the following into the FAQ
this ok?

Reply:
![Uploading IMG_0161.jpeg…]()

Reply:
Neither Influx nor Grafana are NodeJS based, I sincerely doubt they caused a NodeJS update.¨

Also the problem is not really NodeJS v20, instead any update of a major NodeJS version may break things and in practise requires a reinstall of the server and the plugins.

Reply:
noted,
however that wasnt my experience.

after ending up with a higher node version installed, 
and with assistance from scott, i did a reinstall of signalk, 
however that didnt resolve my problem, so i then did a reinstall of node and then signal,
to get back to node 18.

pls modify or delete the faq as appropriate.

Reply:
I removed the section you added as to the best of my knowledge it is incorrect and will now close this PR.

Please continue to be involved in improving our documentation, even if this particular start did not pan out very well!.